config/environments/production.rb

```diff
@@ -8,14 +8,16 @@
   config.assets.js_compressor = :uglifier
   config.assets.compile = false
   config.log_level = :info
-  config.action_mailer.perform_caching = false
   config.i18n.fallbacks = true
   config.active_support.deprecation = :notify
   config.active_record.dump_schema_after_migration = false
   config.public_file_server.headers = {
     "Cache-Control" => "public, max-age=31557600"
   }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { address: "lib-ponyexpr.princeton.edu" }
```

Comment:
Should this value be managed within https://github.com/pulibrary/princeton_ansible?

Reply:
I'm not sure; what are our criteria for that decision? Seems like would be fine in either place. It's in rails config for orangelight.

Reply:
Oh, sorry then, please disregard that comment.